sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,71 @@
+		{ "selector": "meta[name=\"dcterms.creator\"]", "attr": "content", "path": "institutionopt" },
+		{ "selector": "[data-feedback-link]", "attr": "data-feedback-link", "path": "contact/link" },
+		{ "selector": "[data-feedback-url]", "attr": "data-feedback-url", "path": "contact/url" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">Give feedback about this page</h3>
+			<form action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk"}'>
+				<div data-wb-json='{
+					"url": "#[feedback]",
+					"mapping": [
+						{ "selector": "input", "attr": "name", "value": "/@id" },
+						{ "selector": "input", "attr": "value", "value": "/@value" }
+					]
+				}'>
+					<template>
+						<input type="hidden" name="" value="" />
+					</template>
+				</div>
+				<fieldset class="gc-pft-btns chkbxrdio-grp row row-no-gutters d-sm-flex flex-sm-wrap align-items-sm-center">
+					<legend class="col-xs-12 col-sm-7 nojs-col-sm-12 col-md-9 col-lg-8 text-center text-sm-left nojs-text-left mrgn-tp-sm pr-sm-3 nojs-pr-sm-0"><span class="field-name">Did you find what you were looking for?</span></legend>
+					<div class="col-xs-12 nojs-show">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
+						<button class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm mrgn-lft-sm" data-wb-doaction='[
+							{"action":"removeClass","source":".gc-pft-no","class":"nojs-show"},
+							{"action":"addClass","source":".gc-pft-btns","class":"hide"}
+						]'>No</button>
+					</div>
+				</fieldset>
+				<div class="gc-pft-no nojs-show">
+					<p id="why-note" class="nojs-show mrgn-tp-lg mrgn-bttm-md"><strong>If not, tell us why below:</strong></p>
+					<p class="nojs-hide wb-inv"><strong>Tell us why below:</strong></p>
+					<details>
+						<summary><b>Need urgent help with a problem? Contact us</b></summary>
+						<p class="mrgn-bttm-0 mrgn-tp-md">
+							<a href="#" data-wb-json='[
+							{ "url": "#[feedback]/contact/url", "type": "attr", "attr": "href" },
+							{ "url": "#[feedback]/contact/link", "type": "replace" }
+							]'></a>
+						</p>
+					</details>
+					<div aria-live="off">
+						<div class="form-group">
+							<label for="problem" class="mrgn-bttm-0"><span class="field-name">Please provide more details</span></label>
+							<p id="problem-note" class="mrgn-bttm-sm"><small><b>You will not receive a reply. Don't include personal information (telephone, email, SIN, financial, medical, or work details).</b></small></p>
+							<p><small>Maximum 300 characters</small></p>
```

Comment:
Here, the use of `<small>` is it conform to spec or be used as styling?

HTML spec said: `<small>` is for small print
> Small print typically features disclaimers, caveats, legal restrictions, or copyrights. Small print is also sometimes used for attribution, or for satisfying licensing requirements.
> The [small](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element) element should not be used for extended spans of text, such as multiple paragraphs, lists, or sections of text. 
>
> It is only intended for short runs of text. The text of a page listing terms of use, for instance, would not be a suitable candidate for the [small](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element) element: in such a case, the text is not a side comment, it is the main content of the page.
>
> The [small](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element) element must not be used for subheadings; for that purpose, use the [hgroup](https://html.spec.whatwg.org/multipage/sections.html#the-hgroup-element) element.

reference: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element